src/components/BookingPanel/BookingPanel.js

```diff
@@ -89,6 +90,7 @@ const BookingPanel = props => {
     : 'BookingPanel.perUnit';
 
   const classes = classNames(rootClassName || css.root, className);
+  const titleClasses = classNames(css.bookingTitle, titleClassName);
```

Comment:
This just creates a string "BookingPanel__bookingTitle_asdfasd TransactionPage__bookingTitle_werwee"
Which means that both styles come from main.bundle.css and the import order might change the overriding rules.

maybe  change this to`classNames(titleClassName || css.bookingTitle)`

Reply:
And override all the values